composer.json

```diff
@@ -11,12 +11,12 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.4.0",
+        "phpseclib/phpseclib": "^2.0",
```

Comment:
I've reverted the composer.json changes and left any README changes up to you. This way you can mention it how it suits you best.

I'd like to point out though that the phpseclib dependency is currently required to run the old/current spark command, without any note in the README. It could be viable to move the dependency notification up to that readme section?